assets/js/components/CookieOverlay.js

```diff
@@ -12,8 +12,16 @@ module.exports = class CookieOverlay {
     this.$elm = $elm;
     this.window = _window;
     this.doc = doc;
+    try {
+      this.domain = this.window.elifeConfig.domain;
```

Comment:
I think this should be:

```this.domain = this.window.elifeConfig.domain || 'elifesciences.org';``` 

That way, if the `domain` property is available, but has been set to an empty string, the default value will be properly assigned.